src/java.base/share/classes/javax/net/ssl/KeyStoreBuilderParameters.java

```diff
@@ -68,8 +68,7 @@ public KeyStoreBuilderParameters(List<Builder> parameters) {
             throw new IllegalArgumentException();
         }
 
-        this.parameters = Collections.unmodifiableList(
-            new ArrayList<Builder>(parameters));
+        this.parameters = List.copyOf(parameters);
     }
```

Comment:
If you leave this as is, you can use `<>`

Reply:
Using <>.